docs/versioned-plugins/codecs/dots-v3.0.3.asciidoc

```diff
@@ -20,12 +20,5 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
+This codec generates a dot(`.`) to represent each Event it processes. This is typically used with `stdout` output to provide feedback on the terminal. It is also used to measure Logstash's throughtput with the `pv` command.
 
-
-[id="{version}-plugins-{type}s-{plugin}-options"]
```

Comment:
There are no config options, so this whole section needs to be removed--it breaks the book to have a table with no content.

It would be nice to see an example here.